python/ray/data/_internal/execution/operators/map_operator.py

```diff
@@ -257,7 +268,9 @@ def progress_str(self) -> str:
         raise NotImplementedError
 
     def get_metrics(self) -> Dict[str, int]:
-        return self._metrics.to_metrics_dict()
+        # Return a flatten metrics dict.
+        # TODO(Clark): Preserve this metrics hierarchy.
+        return self._metrics.object_store_metrics.to_metrics_dict()
```

Comment:
Only returning the object store metrics for now, we can include the data munging metrics once they have stabilized.

Reply:
I think we can expose both (I don't think we document/promise anything about these metrics in extra metrics!)